tests/test_install.sh

```diff
@@ -1,192 +1,284 @@
 #!/bin/bash
-
 set -ueo pipefail
 
 if [[ "$#" -ne 1 ]]; then
-    echo "Usage: ./test_install.sh [install | uninstall | upgrade]"
-    exit 1
+  echo "##+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+#+"
```

Comment:
Doesn't matter too much since I only consider this an internal test script, but could we at least be consistent, i.e. "#+#" vs "###".

Reply:
Agreed.